src/storefront/api/cart/info.ts

```diff
@@ -0,0 +1,31 @@
+import { IBillingInfo, IShippingInfo } from "@/types/cart";
+
+export const putShippingInfo = async (
+  token: string,
+  shippingInfo: IShippingInfo
+) => {
+  const response = await fetch(`/api/cart/${token}/shipping-info/`, {
+    method: "PUT",
+    headers: {
+      "Content-Type": "application/json",
```

Comment:
I think this isn't neccessary, becaue content type should be already set in axios instance, right?

Reply:
Removed, it's unneccessary